vllm/model_executor/models/qwen2_vl.py

```diff
@@ -705,7 +707,9 @@ def dummy_data_for_qwen2_vl(
     min_pixels: Optional[int] = None,
     max_pixels: Optional[int] = None
 ) -> Tuple[SequenceData, Optional[MultiModalDataDict]]:
-    image_processor = cached_get_image_processor(ctx.model_config.model)
+    mm_processor_kwargs = ctx.model_config.mm_processor_kwargs or {}
+    image_processor = cached_get_image_processor(ctx.model_config.model,
+                                                 **mm_processor_kwargs)
```

Comment:
You should use the `min_pixels` and `max_pixels` arguments directly, since users may override the model config values per input.

Reply:
You're right. Fixed in the latest commit.